src/modules/accounting/types.py

```diff
@@ -424,3 +424,10 @@ class ValidatorStage(Enum):
     PROVEN = 2
     ACTIVATED = 3
     COMPENSATED = 4
+
+
+@dataclass(frozen=True)
+class ValidatorStatus:
+    stage: ValidatorStage
```

Comment:
If this can be none - it should be marked as optional

Reply:
it's not optional, it can be NONE in enum